pom.xml

```diff
@@ -774,74 +774,6 @@
                 </exclusions>
             </dependency>
 
-            <dependency>
-                <groupId>io.netty</groupId>
-                <artifactId>netty-all</artifactId>
-                <version>${netty.version}</version>
-                <exclusions>
-                    <exclusion>
```

Comment:
actually, my vision is to extend this exclusion list instead of declaring everything explicitly. for example, `netty-codec-socks` does not used by Kyuubi, and we should add it into exclusion list

Reply:
emmm, I know, I will try it again. Thanks

Reply:
I just noticed I was wrong on this assumption, apologize for this comment.

The netty jars come from `grpc-netty`, `arrow-memory-netty`, use `netty-bom` and `grpc-bom` to simplify the dependency management.

Reply:
ok, thanks.